doc/content/guides/connecting-gateways/nasysoutdoorgateway/_index.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "NASYS LoRaWAN® Outdoor Gateway"
+description: ""
+weight: 1
+---
+
+NASYS LoRaWAN® Outdoor Gateway is an 8 Channel LoRaWAN gateway, whose technical specifications can be found in [the official product page][1]. This page guides you to connect it to {{% tts %}}.
```

Comment:
Please use an inline-style link to be consistent with other docs.

Reply:
I think the reason to do this was because he links to this page again, so for DRY reasons I think it's fine to do it this way.

Reply:
Removed references section altogether